day-5/ques-5.js

```diff
@@ -0,0 +1,26 @@
+//Problem Statement: dot-sepaWrite a program flattenObject that takes an object with nested properties and returns a flat object with the nested properties transformed into rated properties. The function should handle nested objects and arrays.
+// const obj = {
+//     a: 1,
+//     b: { c: 2, d: [3, 4] }
+//  };
+//  Output: { 'a': 1, 'b.c': 2, 'b.d.0': 3, 'b.d.1': 4 }
+
+const obj = {
```

Comment:
current output is not equal to  expected output

current output => { '.a': 1, '.b.c': 2, '.b.d.0': 3, '.b.d.1': 4 }

Reply:
resolved